src/app/public/modules/radio/radio.component.ts

```diff
@@ -216,6 +216,13 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     }
   }
 
+  /**
+   * Fires when users focus off a radio button.
+   */
+  public get blur(): Observable<undefined> {
```

Comment:
I would just make this a public property, instead of masking the `EventEmitter` with an `Observable` getter.
```
/**
 * @internal
 */
public blur = new Subject<void>();
```